api/v1alpha1/dnsrecord_types_test.go

```diff
@@ -15,15 +15,10 @@ func TestValidate(t *testing.T) {
 		dnsNames []string
 		wantErr  bool
 	}{
-		{
```

Comment:
this was never really testing for invalid domain, the error was because there were no endpoints that matched the roothost.